src/components/Articles/Articles.jsx

```diff
@@ -0,0 +1,41 @@
+import "./Articles.css"
+import { getArticles } from "../../api"
+import { useState } from "react"
+
+function Articles() {
+
+    const [articles, setArticles] = useState([]);
+
+    getArticles().then((fetchedArticles)=> {
```

Comment:
this needs to be inside a useEffect so that it doesn't refresh the articles (and make this request) every time this component re-renders :)